common-core/src/main/java/com/amplifyframework/notifications/pushnotifications/NotificationPayload.kt

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.notifications.pushnotifications
+
+import android.content.Intent
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+open class NotificationContentProvider(val providerName: String) {
+    data class FCM(val from: String?, val content: Map<String, String>) : NotificationContentProvider("FCM")
```

Comment:
Did we still need from?

Reply:
From is not needed. SDK's use is incorrect.